app/forms/schools/on_boarding/access_needs_support.rb

```diff
@@ -8,6 +8,10 @@ class AccessNeedsSupport < Step
       def self.compose(supports_access_needs)
         new supports_access_needs: supports_access_needs
       end
+
+      def supports_access_needs?
+        supports_access_needs
```

Comment:
should this be `!!supports_access_needs` - ensure it returns a boolean, even if its set to nil, or something invalid?